### ISSUE TITLE
[COT] Replace `getAdminLink` for order-edit links in Analytics tables

### ISSUE THREAD

**Issue by xristos3490** [woothemes/woocommerce#34320]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

Currently, we use the `getAdminLink` helper from the `woocommerce/settings` package to create edit-order admin links inside the Analytics report tables. Example: https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-admin/client/homescreen/activity-panel/orders/index.js#L119

### Expected behavior

We need to ensure that when COT is enabled and synced, these links point to the custom order pages.

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by peterfabian** [comment]
Adding a bit of context here based on my research now.

- the getAdminLink is actually [implemented in the Blocks plugin](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/assets/js/settings/shared/utils.ts#L107-L108) and it doesn't do much on its own besides concatenate string passed as the parameter to some default prefix.
- we actually redirect the list-of-orders and edit-order screens, so theoretically speaking, once https://github.com/woocommerce/woocommerce/issues/35074 gets fixed, you shouldn't need to do any changes in your JS code.

tl;dr: it seems we can close this as fixed once we fix #35074, but please let me know if I misunderstood your use case. 

Thanks!

**Comment by vedanshujain** [comment]
We have added the redirect as part of #35074 which also addresses this issue.
